gnc/sub8_perception/nodes/buoy_2d.py

```diff
@@ -42,6 +44,11 @@ def __init__(self):
         # Occasional status publisher
         self.timer = rospy.Timer(rospy.Duration(1.0), self.publish_target_info)
 
+        self.boost = cv2.Boost()
+        rospy.loginfo("Loading boost")
+        self.boost.load('/home/jacob/catkin_ws/src/Sub8/gnc/sub8_perception/sub8_vision_tools/machine_learning/boost_huge.cv2')
```

Comment:
Is this pointing to the correct location to where the boost_huge.cv2 file will be on the sub or just your computer?